tsconfig.json

```diff
@@ -1,12 +1,18 @@
 {
   "compilerOptions": {
-    "target": "es5",
+    "allowSyntheticDefaultImports": true,
+    "declaration": true,
+    "esModuleInterop": true,
```

Comment:
The docs say this:

> Emit `__importStar` and `__importDefault` helpers for runtime babel ecosystem compatibility and enable `--allowSyntheticDefaultImports` for typesystem compatibility.

This implies that `allowSyntheticDefaultImports` is implicit with this option, though it's not clear. Is that true?

Reply:
Nice, I can verify `esModuleInterop` does also turn on `allowSyntheticDefaultImports`, so have removed.